SampleGoGrpc/client.go

```diff
@@ -0,0 +1,116 @@
+package main
```

Comment:
I feel happy-path test is reasonable, I can't really think of adding fuzz testing to it